dev/io.openliberty.security.jakartasec.3.0.internal_fat.common/publish/shared/config/oidcClient/allValues.openIdConfig.properties

```diff
@@ -4,4 +4,4 @@ redirectURI=https://localhost:rp_SecurePort_rp/rp_AppName_rp/Callback
 providerBase=https://localhost:op_SecurePort_op/oidc/endpoint/OP1
 
 callerNameClaim=sub
-callerGroupsClaim=groups
+callerGroupsClaim=groupIds
```

Comment:
This value should have already been fixed/updated - I'm suspecting you're going to have a merge conflict.

Reply:
it should be okay since there's no difference between what i have and what's in integration currently
i.e., they're both `callerGroupsClaim=groupIds`

also, rebased with integration locally and there was no merge conflict